cirq/protocols/measurement_key_protocol.py

```diff
@@ -143,3 +150,8 @@ def is_measurement(val: Any) -> bool:
     a non-empty result for them.
     """
     return bool(measurement_keys(val))
+
+
+def with_measurement_key_mapping(val: Any, key_map: Dict[str, str]):
```

Comment:
nit: Add docstrings to public method exposed to the cirq namespace

Reply:
Done.